### ISSUE TITLE
What's the difference of choosing different Batch Size

### ISSUE THREAD

**Issue by CohenQU** [CohenQU/Artificial-Intelligence-Guide#6]
If we passed our entire training set to the model at once (batch_size=1), then the process we just went over for calculating the loss will occur at the end of each epoch during training.

If we split our training set into batches, and passed batches one at a time to our model, then the loss would be calculated on each batch.

With either method, since the loss depends on the weights, we expect to see the value of the loss change each time the weights are updated. Given that the objective of SGD is to minimize the loss, we want to see our loss decrease as we run more epochs.
